modules/ROOT/assets/attachments/sync-gateway-public.yaml

```diff
@@ -1011,6 +1011,42 @@ paths:
                   $ref: '#/definitions/UserContext'
             401:
               description: Authentication failed. Facebook verification server status 400
+  /{db}/_google:
+    parameters:
+      - $ref: '#/parameters/db'
+    post:
+      tags:
+        - auth
+      summary: User creation / authentication via Google
+      description: |
+        Sync Gateway allows users to authenticate using a Google account. Your application is responsible for generating a Google token; this generally needs to be done by running the Google login flow inside a web-view and capturing the generated token. This endpoint can be used to check the validity of the access token.
+        To allow Google Login with Sync Gateway, it must be explicitely enabled in the Sync Gateway configuration file by setting the [google.register](config-properties.html#facebook-register) property to `true`.
```

Comment:
SG config must also specify `app_client_id` in the config under Google.

Reply:
Addressed in https://github.com/couchbase/docs-sync-gateway/commit/5846a4b9797668b08e4936a0723a72e04f13b053